### PR TITLE
Initiate Github actions

### PR DESCRIPTION
**Pull request by RajithaKumara**
[Note]
Need to set secrets (for the database password):
https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets

But not necessary to make the database password as a secret.

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                    container port: 3306
+                    character set server: 'utf8mb4'
+                    collation server: 'utf8mb4_general_ci'
+                    mariadb version: '10.2.31'
+                    mysql database: 'orangehrm'
+                    mysql root password: ${{ secrets.RootPassword }}
+                    mysql user: 'developer'
+                    mysql password: ${{ secrets.DatabasePassword }}
+
+            -   uses: shivammathur/setup-php@v2
```
Comment (samanthajayasinghe) [inline .github/workflows/php.yml:32]: can't we get via php official repo ?

Comment (RajithaKumara) [reply .github/workflows/php.yml:32]: Do you have any reference? I used [this](https://github.com/marketplace/actions/setup-php-action) one.

**Comment by RajithaKumara** [comment]
@samanthajayasinghe Github actions take more than one hour to finish unit tests ([ref](https://github.com/RajithaKumara/orangehrm/runs/671069965?check_suite_focus=true)). But Travis does not take that much time ([ref](https://travis-ci.org/github/orangehrm/orangehrm)).

**Comment by RajithaKumara** [comment]
Closing since #729 merged

**Pull request closed without merging**